_Sidebar.md

```diff
@@ -14,10 +14,10 @@
 **Developing Oppia**
   * [[FAQs|Frequently-Asked-Questions]]
   * [[Installing Oppia|https://github.com/oppia/oppia/wiki/Installing-Oppia]]
-  * Tutorials
-    * [[Tutorial 1|Tutorial-1]]
   * Getting started with the codebase
     * [[Populating data on local server|Populating-data-on-local-server]]
+    * Tutorials
+      * [[Tutorial 1|Tutorial-1]]
```

Comment:
No one knows what "tutorial 1" is -- give it a name like "Make a simple UI change".

Reply:
Done, Thanks.